weasyl/define.py

```diff
@@ -1081,6 +1088,9 @@ def cdnify_url(url):
 
 
 def get_resource_path(resource):
+    if reload_templates:
```

Comment:
Assets? That might be too specific to non-templates. It looks like pyramid separately allows reloading of 'templates' and 'assets'. Is `reload_templates_and_assets` too verbose?